test/stability/testCksum/testCksum.go

```diff
@@ -100,6 +106,27 @@ func main() {
 	kvmwo := flag.Bool("kvm-workaround", false, "Use this if you run on Qemu/KVM with hardware acceleration which has bug in checksum calculation for TCP packets")
 	flag.Parse()
 
+	if hwol {
+		testCksumCommon.VirtualBoxWorkaround = *vbwo
+		testCksumCommon.KVMWorkaround = *kvmwo
+	}
+	flow.CheckFatal(initTest())
+	flow.CheckFatal(executeTest(testScenario, useIPv4, useIPv6, useUDP, useTCP, useICMP, useVLAN))
+}
+
+func configTest(shouldUseIPv4, shouldUseIPv6, shouldUseUDP, shouldUseTCP, shouldUseICMP, shouldUseVLAN bool) error{
+	useIPv4 = shouldUseIPv4
+	useIPv6 = shouldUseIPv6
+	useICMP = shouldUseICMP
+	useTCP = shouldUseTCP
+	useUDP = shouldUseUDP
+	useVLAN = shouldUseVLAN
+	if hwol {
+		print("hardware cksums and ")
+	}
+	if useVLAN {
+		print("vlan tag and ")
+	}
 	if useIPv4 {
```

Comment:
&& ?

Reply:
it is just for information which configuration is used. This style was used before, I just appended vlan to it.